sections/futures/MarketDetails/MarketDetails.tsx

```diff
@@ -35,22 +37,45 @@ type MarketDetailsProps = {
 	mobile?: boolean;
 };
 
+interface OpenInterestDetailProps extends MarketDetailsProps {
+	isLong: boolean;
+}
+
 const MarketDetails: React.FC<MarketDetailsProps> = ({ mobile }) => {
 	const dispatch = useAppDispatch();
 	const showHistory = useAppSelector(selectShowHistory);
+
+	const MarketDetailsDesktop = () => (
+		<MarketDetailsContainer>
+			<MarketPriceDetail />
+			<IndexPriceDetail />
+			<DailyChangeDetail />
+			<OpenInterestLongDetail />
+			<OpenInterestShortDetail />
+			<MarketSkew />
+			<HourlyFundingDetail />
+		</MarketDetailsContainer>
+	);
+
+	const MarketDetailsMobile = () => (
+		<MarketDetailsContainer mobile={mobile}>
+			<IndexPriceDetail mobile={mobile} />
+			<MarketSkew mobile={mobile} />
+			<HourlyFundingDetail mobile={mobile} />
+			<MarketPriceDetail mobile={mobile} />
+			<DailyChangeDetail mobile={mobile} />
+			<FlexDivRow style={{ columnGap: '25px' }}>
```

Comment:
Prefer to avoid inline styles, could we add this as a prop to FlexDivRow instead maybe?

Reply:
Yes. That's a good idea. Updated.